src/client/apps/edit/components/content/sections/related_articles/index.tsx

```diff
@@ -146,7 +160,17 @@ export class RelatedArticles extends Component {
           color={color}
           onChange={this.onAddArticle}
         />
-      </div>
+      </Flex>
     )
   }
 }
+
+const RelatedArticlePreview = styled.div`
+  div[class^="ArticleCard__ImageContainer"] {
+    min-height: 300px;
```

Comment:
Why add this min-height here? Were these CSS props (background and min-height) defined in a stylesheet under the `RelatedArticles__preview` class previously?

Reply:
This is a placeholder box for when an image is not present -- the empty list shows an empty article that demo's what the related article would look like.